src/v2/views/MonsterCollectionOverlay/MonsterCollectionContent.tsx

```diff
@@ -228,12 +228,15 @@ export function MonsterCollectionContent({
           When you deposit NCG, the monsters go on an expedition to get the
           treasure.
         </DepositDescription>
-        {!isEditing && stakeState && tip && (
-          <DepositDescription>
-            About {getRemain(stakeState.claimableBlockIndex - tip)} of deposit
-            days!
-          </DepositDescription>
-        )}
+        {!isEditing &&
+          stakeState &&
+          tip &&
+          tip > stakeState.claimableBlockIndex && (
```

Comment:
I'm not sure about JavaScript language but in this expression, the tip should be less than `claimableBlockIndex`, to hide the remaining days if it's claimable now.

When the tip is more than `claimableBlockIndex`, it means it's claimable so I thought the component will be shown. `stakeState.claimableBlockIndex - tip` will be always negative.

<img width="266" alt="image" src="https://user-images.githubusercontent.com/26626194/172751476-9a638f00-3b7a-4791-84cd-a636e631cb92.png">

Reply:
You're right it's the opposite